collector/pkg/model/attribute_map.go

```diff
@@ -175,6 +183,14 @@ func (v *stringValue) Type() AttributeValueType {
 	return StringAttributeValueType
 }
 
+func (v stringValue) MarshalJSON() ([]byte, error) {
```

Comment:
Why not use the pointer as the function receiver?

Reply:
Same as above